src/calculateRentalCost.js

```diff
@@ -5,6 +5,21 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_FOR_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const result = PRICE_FOR_DAY * days;
+
+  if (days >= LONG_TERM) {
+    return result - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    return result - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
You should avoid using `else` after an `if` block that contains a `return` statement. An `if` block with a `return` statement will exit the function immediately, making the `else` unnecessary and can be omitted to simplify the code structure.